Client/mods/deathmatch/logic/CClientEntity.cpp

```diff
@@ -303,6 +303,20 @@ CLuaArgument* CClientEntity::GetCustomData(const char* szName, bool bInheritData
     return NULL;
 }
 
+CLuaArguments* CClientEntity::GetAllCustomData(CLuaArguments* table)
+{
+    assert(table);
+
+    // Grab it and return a pointer to the variable
```

Comment:
Comment makes no sense. The `Variable` is a value by itself, so you're passing a value to `PushArgument`, so you copy it.
The `key` is an `SString` so you copy it as well.
Correct me if im wrong.

Reply:
Yeah that's the same code from https://github.com/multitheftauto/mtasa-blue/blob/927509565ab3a8d48df006440654c38e0942aa89/Server/mods/deathmatch/logic/CElement.cpp#L521